addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5369,19 +5369,34 @@ registry.layout_column = SnippetOptionWidget.extend(ColumnLayoutMixin, {
         let numberColumns, numberRows;
```

Comment:
Commit message nitpicks: 
- > [IMP] web_editor, website: open media dialog on image drop

  => `[IMP] web_editor, website: open media dialog on new image addition` 
 Since not only the drop is concerned :wink: 
- > When a user drops a building block "image" or clicks on the "add element" button for "image" in grid mode block,`
 
  => I find it clearer to write this: `When a user drops an "Image" snippet or adds an image in grid mode with the "Add Elements" option,` :slightly_smiling_face: 

- > `When a user is dragging the image building block, instead of using
the "s_text_image" image as the default, we use a new placeholder image
before dropping.

  => `When a user drops an "Image" snippet, a new placeholder image is used instead of the the "Text-Image" one, before choosing a new image in the media dialog.` 
(since technically, it was already dropped so it is before choosing an image, not before dropping) :slightly_smiling_face: